npmpackagejsonlint.config.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+	// See https://www.npmjs.com/package/@wordpress/npm-package-json-lint-config
+	extends: '@wordpress/npm-package-json-lint-config',
+	rules: {
+		'prefer-no-devDependencies': 'error',
+		'prefer-property-order': 'off',
+		'require-bugs': 'off',
+		'require-homepage': 'off',
+		'require-keywords': 'off',
+		'valid-values-author': [ 'error', [ 'Automattic Inc.' ] ],
+		'valid-values-name-scope': [ 'error', [ '@automattic' ] ],
+	},
+	overrides: [
+		{
+			patterns: [
+				'./apps/full-site-editing/package.json',
+				'./apps/wpcom-block-editor/package.json',
+				'./package.json',
+			],
+			rules: {
+				'prefer-no-devDependencies': 'off',
+			},
+		},
+		{
+			patterns: [
+				'./package.json',
+				'./packages/eslint-config-wpcalypso/package.json',
```

Comment:
ditto for these. could we use a * to match all of the packages?

Reply:
I would expect `valid-values-name-scope` rule to be enforced by default and this is the list of packages that are old and didn't follow `@automattic/*` naming convention.

No new packages should be added without `@automattic/*`, right?

Reply:
ahhh, of course. 👍